app/controllers/submit.js

```diff
@@ -350,6 +376,13 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
     }),
 
     actions: {
+        editLicense(license, validates) {
+            this.set('basicsLicense', license);
+            this.set('licenseValid', validates);
+        },
+        applyLicenseToggle(apply) {
+            this.set('applyLicense', apply);
```

Comment:
But you only care about the state of applyLicense though, right? So that case, set applyLicense to true.